staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -164,6 +184,9 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 			klog.Warningf("Failed calling webhook, failing closed %v: %v", hook.Name, err)
 			return apierrors.NewInternalError(err)
 		}
+		if rejectionErr, ok := err.(*webhookutil.ErrWebhookRejection); ok {
+			return rejectionErr.ToStatusErr()
```

Comment:
is there a test ensuring the structured status error from the webhook gets returned from `Admit()`?

Reply:
yes there is: https://github.com/kubernetes/kubernetes/blob/c80d2c228353e78c9b85bb43bddbf5f2917be4ba/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/plugin_test.go#L170-L171
https://github.com/kubernetes/kubernetes/blob/c80d2c228353e78c9b85bb43bddbf5f2917be4ba/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/plugin_test.go#L145-L147
examples of expected error: webhook rejection with 403, error calling webhook and failclose
https://github.com/kubernetes/kubernetes/blob/c80d2c228353e78c9b85bb43bddbf5f2917be4ba/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/testcase.go#L482-L522